package.json

```diff
@@ -2,138 +2,46 @@
   "name": "pillarwallet",
   "version": "0.1.0",
   "private": true,
-  "devDependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/preset-flow": "^7.0.0",
-    "@storybook/addon-actions": "^5.3.12",
-    "@storybook/addon-links": "^5.3.12",
-    "@storybook/addon-ondevice-actions": "^5.3.12",
-    "@storybook/addon-storyshots": "^5.3.12",
-    "@storybook/addons": "^5.3.12",
-    "@storybook/react-native": "^5.3.12",
-    "@storybook/react-native-server": "^5.3.12",
-    "babel-core": "7.0.0-bridge.0",
-    "babel-eslint": "^8.2.2",
-    "babel-jest": "^23.4.2",
-    "babel-loader": "^8.0.6",
-    "babel-plugin-module-resolver": "^3.1.1",
-    "babel-plugin-styled-components": "^1.5.1",
-    "babel-runtime": "7.0.0-beta.3",
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
-    "eslint": "^4.19.1",
-    "eslint-config-airbnb": "^16.1.0",
-    "eslint-import-resolver-babel-module": "5.0.0-beta.0",
-    "eslint-plugin-flowtype": "^2.46.1",
-    "eslint-plugin-flowtype-errors": "^3.5.1",
-    "eslint-plugin-import": "^2.10.0",
-    "eslint-plugin-jest": "^21.15.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-promise": "^3.7.0",
-    "eslint-plugin-react": "^7.7.0",
-    "eslint-plugin-react-native": "^3.2.1",
-    "flow-bin": "0.70.0",
-    "flow-typed": "^2.4.0",
-    "husky": "^1.1.3",
-    "invariant": "^2.2.4",
-    "jest": "^23.4.2",
-    "jest-html-reporter": "2.0.0",
-    "jest-react-native": "^18.0.0",
-    "jest-styled-components": "^5.0.1",
-    "jetifier": "^1.6.4",
-    "jsdom": "^16.1.0",
-    "metro-react-native-babel-preset": "^0.51.1",
-    "mock-socket": "^9.0.3",
-    "npm-run-all": "^4.1.5",
-    "prop-types": "^15.6.1",
-    "react-dom": "^16.3.0-alpha.1",
-    "react-native-debugger-open": "^0.3.19",
-    "react-native-dotenv": "^0.2.0",
-    "react-native-storybook-loader": "^1.8.1",
-    "react-test-renderer": "16.6.3",
-    "reactotron-react-native": "^3.2.2",
```

Comment:
out of curiosity, now that Reactotron is removed what do we use to inspect the Redux actions and state?

Reply:
Redux dev tools are added as enhancer which allows us to inspect it. Reactotron was never used.